<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Eq</class>
 <widget class="QDialog" name="Eq">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>130</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>609</width>
    <height>130</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>609</width>
    <height>130</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Equalizer</string>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>51</y>
     <width>581</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>_________________________________________________________________________________________________</string>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_7">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_9">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_10">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_11">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_12">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_13">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_14">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_15">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_16">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_17">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_18">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_19">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_20">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_21">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_22">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_23">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_24">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_25">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_26">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_27">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_28">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>56</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0dB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>+15dB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>108</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> -15dB</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
